### ISSUE TITLE
Readiness probe failed: /usr/local/bin/status-probe.sh failed check: systemctl -q is-active glusterd.service 

### ISSUE THREAD

**Issue by papanito** [gluster/gluster-containers#157]
I've followed the [setup guide](https://github.com/gluster/gluster-kubernetes/blob/master/docs/setup-guide.md) using the `gk-deploy` script

```bash
./gk-deploy.sh --admin-key xxxx --user-key xxx -v -g
```
This went fine. However, the deployed container fail with

```
Readiness probe failed: /usr/local/bin/status-probe.sh failed check: systemctl -q is-active glusterd.service 
```
The `glusterd.service` on the node is running:

```bash
$ systemctl status glusterd
● glusterd.service - GlusterFS, a clustered file-system server
   Loaded: loaded (/lib/systemd/system/glusterd.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2019-10-19 18:48:25 CEST; 1 day 17h ago
     Docs: man:glusterd(8)
  Process: 939 ExecStart=/usr/sbin/glusterd -p /var/run/glusterd.pid --log-level $LOG_LEVEL $GLUSTERD_
 Main PID: 955 (glusterd)
    Tasks: 10 (limit: 4915)
   Memory: 30.5M
   CGroup: /system.slice/glusterd.service
           └─955 /usr/sbin/glusterd -p /var/run/glusterd.pid --log-level INFO
....
```

**Comment by minhnnhat** [comment]
> I've followed the [setup guide](https://github.com/gluster/gluster-kubernetes/blob/master/docs/setup-guide.md) using the `gk-deploy` script
> 
> ```shell
> ./gk-deploy.sh --admin-key xxxx --user-key xxx -v -g
> ```
> 
> This went fine. However, the deployed container fail with
> 
> ```
> Readiness probe failed: /usr/local/bin/status-probe.sh failed check: systemctl -q is-active glusterd.service 
> ```
> 
> The `glusterd.service` on the node is running:
> 
> ```shell
> $ systemctl status glusterd
> ● glusterd.service - GlusterFS, a clustered file-system server
>    Loaded: loaded (/lib/systemd/system/glusterd.service; enabled; vendor preset: enabled)
>    Active: active (running) since Sat 2019-10-19 18:48:25 CEST; 1 day 17h ago
>      Docs: man:glusterd(8)
>   Process: 939 ExecStart=/usr/sbin/glusterd -p /var/run/glusterd.pid --log-level $LOG_LEVEL $GLUSTERD_
>  Main PID: 955 (glusterd)
>     Tasks: 10 (limit: 4915)
>    Memory: 30.5M
>    CGroup: /system.slice/glusterd.service
>            └─955 /usr/sbin/glusterd -p /var/run/glusterd.pid --log-level INFO
> ....
> ```

Hi @papanito , i met the same problem. Did you solve this ? Thank

**Comment by papanito** [comment]
Not really, as I am anyway looking into another solution for gluster in k8s: [kadalu](https://github.com/kadalu/kadalu) 
> A lightweight Persistent storage solution for Kubernetes / OpenShift using GlusterFS in background. 
There are still some issues bit the guys are very responsive and the solution looks promising - at least for my use case
